### DISCUSSION TITLE
How to switch back to admin user?

### DISCUSSION THREAD

**Discussion by BinaryMan32** [linuxserver/Heimdall#717] [Q&A]
Just started running version 2.2.2 in kubernetes today.

I created a new user and was able to switch back and forth between my new user and the admin user.

However, once I add a password to either the new user or the admin user, I can't switch back to the admin user. Heimdall displays a page for me to choose between admin and the new user, but regardless of which one I pick, I end up at the new user's dashboard. 

Since only the admin user can manage users, once I'm in this state I'm unable to do anything to try to fix the problem except wipe /config and start over.

Any suggestions on how to proceed?

**Comment by Erriez** [comment]
Here is a screencapture to switch between a user and admin:

![Heimdall2](https://user-images.githubusercontent.com/8849873/109385264-e9829500-78f2-11eb-9b7e-2df702b32027.gif)
